add ultralm results

cyl628 opened the pull request.

rtaori commented on results/ultralm-13b/leaderboard.csv:
The leaderboard that should be updated is here https://github.com/tatsu-lab/alpaca_eval/blob/main/src/alpaca_eval/leaderboards/data_AlpacaEval/alpaca_eval_gpt4_leaderboard.csv 

Could you delete this file and submit a diff against this leaderboard instead?

cyl628 replied:
Sorry for the overlook. I have fixed it in the latest commit.

rtaori commented on src/alpaca_eval/models_configs/ultralm-13b/configs.yaml:
```diff
@@ -0,0 +1,14 @@
+    model_kwargs:
+      torch_dtype: 'float16'
+    max_new_tokens: 2048
+    temperature: 0.5
```
is there a particular reason for temp to be 0.5 here? We've kept temp=0.7 for all the other models as default to maintain consistency, so we'd prefer to keep that if possible.

cyl628 commented:
Hi, thanks for your information.
There is no particular reason for temp=0.5, just our default setting. We tried temp=0.7 and the results turned out surprisingly good. I have updated the results with temp=0.7 and the target leaderboard., and deleted unnecessary files.

Let me know if there is any problem.
Many thanks!

rtaori commented:
great! just have minor configs comments left

rtaori commented:
Great! thanks for your contribution :)
